packet.go

```diff
@@ -48,10 +46,15 @@ func (p *packet) getStatusCode() int {
 }
 
 func (p *packet) getError() error {
+	ERROR_DESCRIPTIONS := map[int]string{
+		2:   "Invalid expression",
+		3:   "Parse error",
+		127: "Unknown variable/method"}
+
 	if p.err != nil {
 		return p.err
 	}
-	return errors.New("Command error with status: " + strconv.Itoa(p.getStatusCode()))
+	return errors.New("Command error with status: " + ERROR_DESCRIPTIONS[p.getStatusCode()])
```

Comment:
If there isn't a description for a particular status code this will return nil. Maybe check whether a description exists, if not return the status code.